### PR TITLE
Notification page

### PR DESCRIPTION
**Pull request by corlard3y**
1. **Error message on Verify Address page - Completed.**
2. Welcome Back Page for existing users, with a Link to the DAPP that redirects users to Opt-In to a Channel. (Can also guide them to docs or relevant videos, if need be) - Completed.
3. Update HomePage UI - Completed
4. Validate Correct Wallet Address - Completed
5. Update Wallet Address Page UI - Completed
6. Notify successful signup - Completed
7. Add animations and transition effects to the UI using GSAP - Completed 
8. Top Bar on extension - Completed
-Cancel button on the extension.
-EPNS Logo on the extension
9. Update notification page UI- Completed
10. Notification page illustration for new users - Completed
11. Switch account dropdown and other options to be added - Completed
12. Inbox/Spam notification - Completed

QA Observations

- Change cursor on both cancel buttons and regular buttons -DONE
- Brave Browser (size)- DONE
- Address Page remove red line in input text field -DONE
- EPNS Logo Tool-Tip -DONE
- Check dimension in extension - DONE
- Wallet Address Text Colour - DONE
- Circular border on cancel button -DONE
- Extension not opening beyond home page in Brave browser - DONE (Users need to check their settings, under ‘privacy & security’ and Enable ‘**Use Google services for push messaging**’ and relaunch their brave browser to be able to access the full functionality of the browser on brave browser)

### CODE REVIEW

File: src/components/Transitions/Transitions.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        tl.to(".transition-effect", {
+            duration: 5,
+            x: 2600,
+            ease: "power4.easeOut"
+        })
+    },[])
+
+
+    return ( 
+        <div>
```
Comment (subhranshudas) [inline src/components/Transitions/Transitions.js:18]: i think we can remove the extra outer `div`

File: src/components/Transitions/Transitions3.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  useEffect(() => {
+    tl.to(".transition3-effect", {
+      duration: 5,
+      x: 2600,
+      ease: "power4.easeOut",
+    });
+  }, []);
+
+  return (
+    <div>
```
Comment (subhranshudas) [inline src/components/Transitions/Transitions3.js:17]: i think we can remove the extra outer `div`

**Review by subhranshudas — COMMENTED** [review]
Hey, Kolade please check on this comments!

File: src/components/AddressPage/AddressPage.js [hunk 3]
```diff
@@ -8,91 +9,199 @@ import {
...
+      setErrorMessage({ message: "Please, input a valid wallet address!" });
+      setDisabled(true);
+      // document.getElementById("input-type").style.border =
+      //   "1.5px solid #f87171";
+    }
+  };
+
+  // wallet address validator
+  const submitAddress = () => {
+    var WAValidator = require("wallet-address-validator");
```
[the remark this answers is not in the document]
Comment (corlard3y) [reply src/components/AddressPage/AddressPage.js:98]: Using required it the means to get the package in the component. import does not work. Changed the var to let

File: src/components/HomePage/HomePage.js [hunk 4]
```diff
@@ -1,91 +1,121 @@
...
+      });
+  }, []);
+
+  useEffect(() => {
+    getToken().then((res) => {
+      setToken(res);
+    });
+  }, []);
 
   const toggle = () => {
```
[the remark this answers is not in the document]
Comment (corlard3y) [reply src/components/HomePage/HomePage.js:66]: It is actually being used. A firebase action is called once you open the extension which generates the token, which is used as a property for API requests in the extension. I know the extension will not work if this token is not used. Probably the BE can explain about the token more

[the remark this answers is not in the document]
Comment (subhranshudas) [reply src/components/HomePage/HomePage.js:66]: are you sure `toggle` is used anywhere?
Screen Shot 2022-05-12 at 9 54 40 AM

File: src/components/AddressPage/AddressPage.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (subhranshudas) [reply src/components/AddressPage/AddressPage.js]: are you sure we can't use `import` ?
https://stackblitz.com/edit/react-code-sample-cbceql?file=index.js

[the remark this answers is not in the document]
Comment (corlard3y) [reply src/components/AddressPage/AddressPage.js]: Thanks. Done and Resolved

File: src/components/HomePage/HomePage.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (corlard3y) [reply src/components/HomePage/HomePage.js:66]: Ohh the toggle function, will remove that

**Review by subhranshudas — APPROVED** [review]
LGTM

**Pull request merged**